modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowOperationTagUtil.java

```diff
@@ -54,7 +55,8 @@ private WorkflowOperationTagUtil() {
   private static final String MINUS = "-";
 
   public static TagDiff createTagDiff(final String tagList) {
-    final String[] targetTags = StringUtils.split(tagList, ",");
+    final ArrayList<String> targetTags = new ArrayList<>();
+    ((ArrayList) targetTags).addAll(Collections.toList(StringUtils.split(tagList, ",")));
```

Comment:
To avoid unchecked assignments and Opencast's weird internal collection utils, maybe use [nullToEmpty(…)](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/ArrayUtils.html#nullToEmpty-java.lang.String:A-) from Apache commons lang instead? The library is included anyway:

```java
final String[] targetTags = ArrayUtils.nullToEmpty(StringUtils.split(tagList, ","));
```